lib/workers/repository/update/branch/index.ts

```diff
@@ -147,6 +147,7 @@ export async function processBranch(
   }
   const keepUpdatedLabel = config.keepUpdatedLabel;
   const artifactErrorTopic = emojify(':warning: Artifact update problem');
+  const artifactNoticeTopic = emojify(':warning: Artifact update notice');
```

Comment:
```suggestion
  const artifactNoticeTopic = emojify(':information_source: Artifact update notice');
```

Please check how this one looks